app/controllers/admin/admin_users_controller.rb

```diff
@@ -67,9 +67,9 @@ def admin_user_params
     end
 
     def authorize_user!
-      unless super_admin_user? @current_user
-        raise ApplicationController::NotAuthorized
-      end
+      return if super_admin_user? @current_user
+
+      raise ApplicationController::NotAuthorized
```

Comment:
The place where I made a manual fix. Other problems were auto resolved by `bundle exec rubocop -a`